components/badges.html

```diff
@@ -0,0 +1,78 @@
+                 </a>
+                 </div> 
+                 <div class="nav-top hidden-sm">
+                    <div class="nav-float-right"><a href = "/index.html" class="hover-effect">home</a></div>
+                    <div class="nav-float-right"><a href = "/documentation.html" class="hover-effect">documentation</a></div>
+                    <div class="nav-float-right"><a href = "https://github.com/sneha-pujari/Component-Library" class="hover-effect">github</a></div>
+                </div>
+    
+            </div>
+        </div>
+        </header>  
+            <nav>
+                <h2>Getting started</h2>
+                <ul>
+                  <li><a class="link-style" href="/documentation.html">Introduction</a></li>
+                </ul>
+                <h2>Components</h2>
+                <ul>
+                    <li><a class="link-style" href="avatar.html">Avatar</a></li>
+                    <li><a class="link-style" href="alerts.html">Alerts</a></li>
+                    <li><a class="link-style" href="badges.html">Badges</a></li>
+                    <li><a class="link-style" href="buttons.html">Buttons</a></li>
+                    <li><a class="link-style" href="cards.html">Cards</a></li>
+                    <li><a class="link-style" href="images.html">Images</a></li>
+                    <li><a class="link-style" href="input.html">Input</a></li>
+                    <li><a class="link-style" href="list.html">List</a></li>
+                    <li><a class="link-style" href="typography.html">Typography</a></li>
+                </ul>
+              </nav>
+              <main>
+                <h2 style="font-size:35px">Badges</h2>
+                <section class="container">
+                    <p></p>
+                    <div class="darkbg">
+                        <h3>Badges on avatars</h3>
+        <div>
+        <img class = "badge-avatar" src = "https://images.unsplash.com/photo-1544568100-847a948585b9?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=967&q=80" alt = "avatar-small"></img>
+        <div class="badge-icon"></div>
+        </div>
+        <h3>Pill badges</h3>
+        <div class="pill-primary">Primary Pill </div>
+        <div class="pill-secondary">Secondary Pill </div>   
+                    </div>
+            <div class="code-display">
+                <code>
+                    &lt;h3>Badges on avatars&lt;/h3> <br />
+                    &lt;div> <br />
+                    &lt;img class = "badge-avatar" src = "https://bit.ly/2Om0Udn" alt = "avatar-small" /><br />
+                    &lt;div class="badge-icon">&lt;/div> <br />
+                    &lt;/div>
+                    &lt;h3>Pill badges&lt;/h3> <br />
+                    &lt;div class="pill-primary">Primary Pill &lt;/div> <br />
+                    &lt;div class="pill-secondary">Secondary Pill &lt;/div> <br />
+
+                </code>

```

Comment:
There is a too-large bottom margin in the badge component page. Pls correct it

Reply:
okay, done!